hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/8_6_0/7296-bulk-export-fails-with-string-resource-ids.yaml

```
@@ -0,0 +1,6 @@
+---
+type: fix
+issue: 7296
+jira: SMILE-7961
+title: "Fixed an issue in bulk export with _includeHistory=true where history records failed to export for resources with client-assigned string IDs. History
+   is now gathered using PIDs rather than forced IDs."
```

```suggestion
title: "Fixed an issue in bulk export with _includeHistory=true where history records failed to export for resources with client-assigned string IDs. History
   is now gathered using PIDs rather than forced IDs."
```
Suggestion: 
- Removed the (unrelated) Jira issue 
- Added some code formatting